src/templates/base.html

```diff
@@ -5,9 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     {% block head %}
       <title>{% block title %}The Web Almanac{% endblock %}</title>
-      <link rel="stylesheet" href="/static/css/normalize.css?v=20200630133215">
+      <link rel="stylesheet" href="/static/css/normalize.css?v={{ get_file_date_info('static/css/normalize.css', 'hash') }}">
```

Comment:
The filename is duplicated in the `href` attr and `get_file_date_info` arg. Could we create a function like this to handle the versioning?

```py
def get_versioned_filename(path):
  version = get_file_date_info(path, 'hash')
  return '/%s?v=%s' % (path, version)
```

_Side note: why isn't the leading slash included in the config keys?_

That would make it possible to do this:

```suggestion
      <link rel="stylesheet" href="{{ get_versioned_filename('static/css/normalize.css') }}">
```

Reply:
Good call.

No reason for no leading slash other than the fact they are relative paths to from where the generate script is called. Have added leading slash to the `static` ones as confusing to web developers if not there. I've left it off templates as they are relative to template dir.